vulnerabilities/importers/apache_httpd.py

```diff
@@ -49,47 +53,58 @@ def updated_advisories(self):
         # skips processing the response further to avoid duplicate work
 
         if create_etag(data_src=self, url=self.url, etag_key="ETag"):
-            data = fetch_xml(self.url)
-            advisories = to_advisories(data)
+            links = fetch_links(self.url)
+            advisories = []
+            for link in links:
+                data = requests.get(link).json()
+                advisories.append(self.to_advisory(data))
             return self.batch_advisories(advisories)
 
         return []
 
-
-def to_advisories(data):
-    advisories = []
-    for issue in data:
+    def to_advisory(self, data):
+        cve = data["CVE_data_meta"]["ID"]
+        description = data["description"]["description_data"]
+        summary = next((item["value"] for item in description if item["lang"] == "eng"), "")
```

Comment:
Using next here is a little cryptic. Would there be a more readable and explicit way to do this?
I reckon that `description` has this shape (and are you sure this is always present?):

```JSON
  "description": {
    "description_data": [
      {
        "lang": "eng",
        "value": "For configurations using proxying...."
      }
    ]
  }
```
so what about this?
```
descriptions = data.get("description", {}).get("description_data", [])
description = None
for desc in descriptions:
    if desc.get("lang") == "eng":
        description = desc.get("value")
        break
```

Reply:
`description` is always present, @AmitGupta7580  checked it.